python/port/mod/turtle/modturtle.cpp

```diff
@@ -131,37 +133,63 @@ mp_obj_t modturtle_isdown() {
 }
 
 mp_obj_t modturtle_pencolor(size_t n_args, const mp_obj_t *args) {
-  if (n_args == 0) {
-    // pencolor()
-    KDColor c = sTurtle.color();
-    mp_obj_t mp_col[3];
-    mp_col[0] = mp_obj_new_int_from_uint(c.red());
-    mp_col[1] = mp_obj_new_int_from_uint(c.green());
-    mp_col[2] = mp_obj_new_int_from_uint(c.blue());
-    return mp_obj_new_tuple(3, mp_col);
+  switch(n_args){
+    case 0:{
+      // pencolor()
+      KDColor c = sTurtle.color();
+      mp_obj_t mp_col[3];
+      if(sTurtle.colorMode() == MicroPython::ColorParser::ColorModes::MaxIntensity255){
```

Comment:
You could create a helper `ColorTuple(args[0], args[1],args[2])` (in order to use it in case 3